packages/rules/docs/rules/strict-css-modules-names.md

```diff
@@ -0,0 +1,32 @@
+# CSS Modules should have the same name as a component and located in the same folder (strict-css-modules-name)
+
+CSS modules should have the same name as the component they are used in.
+
+## Rule Details
+
+This rules prevents the import of a css modules that isn't for this component.
```

Comment:
This rules prevents the import of a CSS modules that isn't for this component.

To re-use CSS, consider using global styles instead.